crates/subspace-farmer/src/commitments.rs

```diff
@@ -123,7 +123,7 @@ impl Commitments {
                 let pieces_to_process = (batch_start + BATCH_SIZE).min(piece_count) - batch_start;
                 // TODO: Read next batch while creating tags for the previous one for faster
                 //  recommitment.
-                let pieces = plot
+                let (pieces, indexes) = plot
```

Comment:
First of all, better call this `piece_indexes` to be more specific. Regardless, I think this should actually be piece offset since we don't care about piece index during commitment creation, so no need to read it, what we want after finding a tag is to read it from disk, after which we can also map offset to piece index.